### ISSUE TITLE
Alphabetically order the "prov/state" list in README for all countries.

### ISSUE THREAD

**Issue by sugatoray** [dr-prodigy/python-holidays#484]
Currently the following four countries' `prov` lists in the README file are not ordered alphabetically. It will be helpful to have them alphabetically ordered for the users of this library.

- India
- Brazil
- Germany
- NewZealand

```rst
=================== ========= =============================================================
Country             ISO code  Provinces/States Available
=================== ========= =============================================================
Brazil              BR/BRA    state = AC, AL, AP, AM, BA, CE, DF, ES, GO, MA, MT, MS, MG,
                              PA, PB, PE, PI, RJ, RN, RS, RO, RR, SC, SP, SE, TO
Germany             DE/DEU    prov = BW, BY, BYP, BE, BB, HB, HH, HE, MV, NI, NW, RP, SL,
                              SN, ST, SH, TH
India               IN/IND    prov = AS, SK, CG, KA, GJ, BR, RJ, OD, TN, AP, WB, KL, HR,
                              MH, MP, UP, UK, TN
NewZealand          NZ/NZL    prov = NTL, AUK, TKI, HKB, WGN, MBH, NSN, CAN, STC, WTL,
                              OTA, STL, CIT
```

**Comment by sugatoray** [comment]
I have already implemented this and will send a PR. So, please assign this to me (@sugatoray). Thank you.

**Comment by dr-prodigy** [comment]
Closed through #485 .. thank you! 👍
